### PR TITLE
Adding Archer Playbook

### PR DESCRIPTION
**Pull request by maiorrulz**

### CODE REVIEW

File: Playbooks/playbook-Archer_Playbook.yml [hunk 1]
```diff
@@ -0,0 +1,341 @@
+id: 8d19cd76-5546-4435-8d86-91baa5ae315c
```
Comment (meirwah) [inline Playbooks/playbook-Archer_Playbook.yml]: the location of the playbook is wrong, should be under TestPlaybooks folder, for instance look at:
https://github.com/demisto/content/blob/master/TestPlaybooks/playbook-urlscan.yml

**Comment by meirwah** [comment]
@maiorrulz did you add archer configuration to the conf file?

**Comment by meirwah** [comment]
@maiorrulz can you attach a screenshot of the playbook?

**Comment by meirwah** [comment]
@maiorrulz build fails

**Comment by maiorrulz** [comment]
download

**Comment by meirwah** [comment]
@maiorrulz it seems you moved the wrong file:
image

you moved the integration , not the test playbook!

**Comment by meirwah** [comment]
@maiorrulz also you need to add the playbook to thew conf file:
https://github.com/demisto/content/blob/master/Tests/conf.json

File: Tests/conf.json [hunk 2]
```diff
@@ -27,6 +27,10 @@
             "timeout": 60,
             "name": "OpenPhish",
             "playbookID": "OpenPhish Test Playbook"
+        },
+        {
+            "name": "RSA Archer",
+            "playbookID": "8d19cd76-5546-4435-8d86-91baa5ae315c"
```
Comment (meirwah) [inline Tests/conf.json]: playbookID should be something like "RSAArcherTestPlaybook", not uuid

**Pull request merged**